worker/codalabworker/aws_batch.py

```diff
@@ -518,6 +536,19 @@ def update_period(self):
     def kill(self, reason):
         self._batch_client.terminate_job(jobId=self.job_id, reason=reason)
 
+    def keep_alive(self):
+        # We call resume periodically as part of the loop while a job is running.
+        # This ensures that if the job transitions to the WORKER_OFFLINE state in the master,
+        # it is transitioned back to running.
+        message = {
+            'hostname': socket.gethostname(),
+            'start_time': current_time(),
```

Comment:
🐑 In the docker version, this is the originally-set start time, not the current time. Should we mimic that behavior? (It actually looks unused in resume_bundle so maybe it doesn't matter, but at least a comment would be good if so?)